dlr/helpers.py

```diff
@@ -309,29 +313,25 @@ def round_voltage(voltage):
 
 
 def read_lines(
-    fpath: str = paths.hifld,
+    fpath: str = paths.lines,
     line_idx_range: slice | None = None,
     crs: str = 'ESRI:102008',
 ):
-    if line_idx_range is None:
-        dflines = gpd.read_file(fpath)
-    else:
-        dflines = gpd.read_file(fpath, rows=line_idx_range)
-
-    if str(dflines.crs) != crs:
-        dflines = dflines.to_crs(crs)
-
+    match fpath:
+        case _ if fpath.endswith('.kml'):
+            fiona.supported_drivers['KML'] = 'rw'
+            dflines = gpd.read_file(fpath, driver='KML', rows=line_idx_range)
+        case _:
+            dflines = gpd.read_file(fpath, rows=line_idx_range)
```

Comment:
When there are only two options I personally find it easier to read to just use `if`/`else`. Is that ok or do you think it's better to always use `match`/`case`?

```suggestion
    if fpath.endswith('.kml'):
        fiona.supported_drivers['KML'] = 'rw'
        dflines = gpd.read_file(fpath, driver='KML', rows=line_idx_range)
    else:
        dflines = gpd.read_file(fpath, rows=line_idx_range)
```